apps/api/src/app/events/services/workflow-queue/trigger-handler-queue.service.ts

```diff
@@ -20,18 +21,18 @@ export class TriggerHandlerQueueService {
       tls: process.env.REDIS_TLS as ConnectionOptions,
     },
   };
-  public readonly queue: Queue;
-  public readonly worker: Worker;
+  private readonly bullmqService: BullmqService;
```

Comment:
The branding they use is BullMQ, so I'd go for: `bullMQService` or `bullMqService`.